include/dlpack/dlpack.h

```diff
@@ -157,6 +157,20 @@ typedef enum {
   kDLComplex = 5U,
   /*! \brief boolean */
   kDLBool = 6U,
+  /*! \brief FP8 data types. Must set DLDataType.bits=8. */
+  kDLFloat8_e3m4 = 7U,
+  kDLFloat8_e4m3 = 8U,
+  kDLFloat8_e4m3b11fnuz = 9U,
+  kDLFloat8_e4m3fn = 10U,
+  kDLFloat8_e4m3fnuz = 11U,
+  kDLFloat8_e5m2 = 12U,
+  kDLFloat8_e5m2fnuz = 13U,
+  kDLFloat8_e8m0fnu = 14U,
```

Comment:
Should we reserve some places between the different float widths in case there are more variants that come up? I didn't get the sense that everyone converged on the format yet given so many variants.

Reply:
giveb enum is not ordered, i think we can go with the common ones and add more in future numbers

Reply:
I thought about that too but unfortunately we only have 256 slots due to the use of `uint8_t` storage in `DLDataType`, so I tried to be cautious about leaving holes.